src/themes/shared/index.ts

```diff
@@ -8,5 +8,6 @@ export { default as fontSizes } from './fontSizes';
 export { default as lineHeights } from './lineHeights';
 export { default as typography } from './typography';
 export { default as opacity } from './opacity';
-export { default as breakpoints } from './breakpoints';
+export { breakpoints } from './breakpoints';
```

Comment:
Could you align the approach for all the exports? Either let's use default or named everywhere. I don't have a strong opinion but let's do it consistently

Reply:
Went with named exports so we can just spread shared in themes and don't need to add every shared token sepearately